lib/browser/api/extensions.js

```diff
@@ -259,6 +259,25 @@ const createTab = function (createProperties, cb) {
   }
 }
 
+const duplicateTab = function (tabId, cb) {
+  try {
+    let tab = getWebContentsForTab(tabId)
+    // XXX What to do if not there?
+    tab._clone(
```

Comment:
this should use `clone` instead of `_clone`. `_clone` is an internal method that is only meant to be used by web-contents.js